### ISSUE TITLE
Cant load LAZ from cloud(Onedrive-sharepoint)

### ISSUE THREAD

**Issue by APOS80** [qgis/QGIS#55551]
### What is the bug or the crash?

Untwine cant find path.

Works when file is on C:

### Steps to reproduce the issue

Load LAZ from a cloud location like sharepoint.

### Versions

QGIS version
3.34.0-Prizren
QGIS code revision
ffbdd67881
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.7.2
PROJ version
9.3.0
EPSG Registry database version
v10.094 (2023-08-08)
GEOS version
3.12.0-CAPI-1.18.0
SQLite version
3.41.1
PDAL version
2.5.5
PostgreSQL client version
15.2
SpatiaLite version
5.1.0
QWT version
6.1.6
QScintilla2 version
2.13.4
OS version
Windows 10 Version 2009

Active Python plugins
cartography_tools
1.2.1
cigeoe_node_tool
1.0
drone_path
0.2
GeometryShapes
0.7
GroupStats
2.2.7
HCMGIS
23.11.23
kmltools
3.1.31
LandSurveyCodesImport
v0.3.0
layer_grid_view
0.1
lftools
1.12.4
nominatim
1.4.5
openlayers_plugin
2.0.0
OpenTopography-DEM-Downloader
2.0
osminfo
0.8.1
processing_r
4.0.0
profiletool
4.2.6
qad
3.0.4
QgisGLViewer
1.4
qgis_hub_plugin
0.1.2
qgis_resource_sharing
1.0.0
QuickOSM
2.2.3
rvt-qgis
0.9.6
SelectionFilter
1.0
spline
0.2
splitmultipart
1.0.0
valuetool
3.0.17
Video_UAV_Tracker
2.1
volume_calculation_tool
0.4
db_manager
0.1.20
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by APOS80** [comment]
Same problem in QGis 3.34.1

![image](https://github.com/qgis/QGIS/assets/6387252/2c0ede9a-ab1d-400a-82af-2d46e53e89de)

**Comment by APOS80** [comment]
> Do you have write permission to the folder? Can you open a laz file which indexing (.copc extension) had already been generated from that folder?

Yes, I use the folder and others every day. I can open the LAZ in CloudCompare frome that folder.

I placed the file on my Jottacloud but i get the same result.

**Comment by radim-ek** [comment]
I found a problem if you have a "ěščřžýáí" in a path LAZ not open. If you open from path without diacritics all is OK.
OK: D:\SomePath\Without\something.laz
FAIL: D:\SomePath\With_a_ěščřžýáí\something.laz

**Comment by baswein** [comment]
Does this pull request fix your issue? It did for me with .laz on google drive folders. https://github.com/qgis/QGIS/pull/62492
